ios/FluentUI/Controls/Button.swift

```diff
@@ -212,6 +281,47 @@ open class Button: UIButton {
         }
     }
 
+    private func normalTitleAndImageColor(for window: UIWindow) -> UIColor {
+        return style == .primaryFilled ? Colors.Button.titleWithFilledBackground : Colors.primary(for: window)
+    }
+
+    private func highlightedTitleAndImageColor(for window: UIWindow) -> UIColor {
+        return style == .primaryFilled ? normalTitleAndImageColor(for: window) : Colors.primaryTint20(for: window)
+    }
+
+    private func disabledTitleAndImageColor(for window: UIWindow) -> UIColor {
+        return style == .primaryFilled ? normalTitleAndImageColor(for: window) : Colors.primaryTint20(for: window)
+    }
+
+    private var normalImageTintColor: UIColor?
```

Comment:
Are you talking about the disabledTitleAndImageColor or normalImageTintColor?
**disabledTitleAndImageColor** can't be cached because there are different variants based on the window.
**normalImageTintColor** is cached as a significant optimization to avoid recreating the tinted image each time the control updates. We only recreate the tinted image when the tint changes. We don't cache the tinted images because that would take up a lot of memory and the tint color depends on the window so it could change.